spec/features/idv/in_person_spec.rb

```diff
@@ -109,6 +109,30 @@
       visit_idp_from_sp_with_ial2(:oidc)
       expect(page).to have_current_path(idv_in_person_ready_to_verify_path)
     end
+
+    it 'handles profiles in fraud review after 30 days', allow_browser_log: true do
+      sign_in_and_2fa_user(user)
+      begin_in_person_proofing(user)
+      complete_prepare_step(user)
+      complete_location_step
+      complete_state_id_step(user)
+
+      # ssn page
+      select 'Review', from: :mock_profiling_result
+      complete_ssn_step(user)
+      complete_verify_step(user)
+      complete_phone_step(user)
+      complete_enter_password_step(user)
+      acknowledge_and_confirm_personal_key
```

Comment:
A pattern I really like is the [`flow_policy_helper`](https://github.com/18F/identity-idp/blob/main/spec/support/flow_policy_helper.rb)

It would be nice to have the flexibility to mock up to a particular step in the flow.

Reply:
yes i had that same thought and was hoping it would be one of the changes that would come out of the audit ticket (the one that includes the helper files)